client/src/components/latestCampaigns/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Typography, Anchor } from 'antd';
+import Campaign from '../common/campaignCard';
+import './style.less';
+
+const { Link } = Anchor;
+const { Title, Text } = Typography;
+function latestCampaigns({ campaigns }) {
```

Comment:
From where do you get the campaigns?

Reply:
i'll do a fetch request using farah's route /capnaigns/?limit=3 but further work needs to be done when we merge the route